### DISCUSSION TITLE
Static map image with live paths etc. on top

### DISCUSSION THREAD

**Discussion by StephanSchuster** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#359] [Q&A] [answered]
Hi. I am new here and overwhelmed by the feature set of this component and all the surrounding work you put into this topic @PiotrMachowski. Thank you.

One question: Is it possible to have a static map image (actually 2: ground floor + 1st floor) but at the same time draw all the live stuff from map extractor like path, vacuum position, etc. on top? To my understanding this is currently not possible, right? Any chance for such a feature?

**Comment by PiotrMachowski** [comment, marked as answer]
It's not possible at this moment, it has to be added to map extractor to work correctly. I plan to do it someday :)

**Reply by StephanSchuster** [reply]
Thanks for the immediate answer. I think this could be useful and combine the best of both worlds.

**Reply by LLlypukk** [reply]
Please, add this information to the top of description of the card functionality. It will save a lot of time for people. I've lost two days to set up my custom map picture and calibration to get almost nothing as result.

**Reply by StephanSchuster** [reply]
@PiotrMachowski: Is there any update or plan on this feature? Just asking ...

**Comment by Photelegy** [comment]
https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/discussions/389#discussion-4032195
